docs/semgrep-app/integrations.md

```diff
@@ -1,19 +1,163 @@
+| Tool | Tier availability |
+| ---- | ---------------- |
+| Slack | Community (Free) |
+| Email | Community (Free) |
+| Jira | Team/Enterprise |
+| Amazon S3 | Team/Enterprise |
+| Webhook | Team/Enterprise |
+
+## Finding available integrations
+
+To find available integrations for Semgrep App, follow these steps:
+
+1. Sign in to your [Semgrep App account](https://semgrep.dev/).
+2. Click **Settings**.
+3. Click **Integrations**.
+
+![Screenshot of Semgrep's "Create New Integration Channel" menu](../img/integration-firstview.png)<br />
+
+## Managing integrations
+
+To view, disable, or enable your saved integration channels:
+
+1. In the **Integrations** tab, click the **gear** icon within the **Rule board**.
+2. Select the toggles to turn notifications on or off for each channel.
+
+![Screenshot of the Semgrep rule board with integrations](../img/integration-ruleboard.png)<br />
+
+## Integrating various third-party tools
+
+This section describes how to integrate Semgrep App into particular third-party tools.
+
+### Slack
+
+Integrating with Slack allows Semgrep to send findings to a channel on your Slack workspace.
+
+To set it up, create an app within Slack and enable that app's **Incoming Webhooks** feature. By creating a webhook, you'll have a URL that serves as Semgrep's endpoint to your Slack workplace. Follow the guide below to create the app and webook, and integrate with Semgrep:
+
+1. Log in to your Slack account. Enter your login details in [Slack workspace](https://slack.com/workspace-signin).
+2. Create an app in your Slack workplace by following this link: [Slack apps](https://api.slack.com/apps?new_app=1).
+3. Select **From scratch** and fill in the fields for your app's name and workspace.
+![Screenshot of a Slack form to fill out new app details](../img/slack-create-app-name.png)
+4. You will be redirected to your new app's settings page. Alternatively, you can find this page by viewing your apps [here](https://api.slack.com/apps) and selecting your integration app from the menu.
+5. Click on **Incoming Webhooks** on the left menu, and then enable the toggle.
+6. After you have enabled **Incoming Webhooks**, follow one of these steps:
+    - Generate a **Webhook URL**.
+    - Request to add a new webhook from a workspace owner.
+7. After you've obtained the Webhook URL, return to Semgrep and create a new Slack integration. Name the integration and copy-paste the URL into the **Slack URL** field. You may opt to select the **Inventory** flag to include notifications about Code Asset Inventory findings.
+8. You can **Test** then select which channel you'd like Semgrep to start posting to.
+![Screenshot of the Slack Request Webhook  ](../img/slack-integration-hooktest.png)<br />
+9. After you've verified that everything is working, remember to **Save** the final configuration.
+10. Remember to turn notifications on by going to the **Rule board**, clicking on the **gear icon,** then clicking on the toggle.
```

Comment:
```suggestion
10. To enable notifications, go to the **Rule board**, click on the **gear icon**, and then click on the integration's toggle.
```

Is it the integration's toggle or integrations toggle? Should this word be bold as well? (I know integrations is not mentioned here - but do you mean the icon of the particular integration? Maybe specify it. Like a Slack or Amazon icon?)
Use imperative as much as possible. It is quicker to read and readers understand what they need to do quicker. Again, please, do not use bold tags or any other tags for the characters which are not part of the words you want to emphasize.

Reply:
Shortened this by saying "Turn notifications on by going to the **Rule board**, clicking on the **gear icon**, then clicking the toggle next to the name of the integration.  (And adjusted all sections accordingly.)